### PR TITLE
accept a config file to map org names to rooms

### LINKED ISSUE #2: accept a configuration file to map orgs to channels
e.g. a deploy within `cap` should send a message in the `cap` channel. Example:

https://github.com/atmos/hubot-deploy/blob/master/docs/config-file.md

### PR DESCRIPTION
**Pull request by afeld**
Closes #2.

See the addition to the README, and let me know if it's unclear! Tested with https://github.com/18F/18f-bot/pull/16, which [posted to Slack](https://18f.slack.com/archives/transient/p1433232557000043) in the correct channel :tada:

### CODE REVIEW

**Comment by mbland** [comment]
So this is my first time looking at CoffeeScript, and it's very cool! Thanks for that. The code under test looks good to me.

That said, while I'm not saying "change the test", I'm wary of stubbing methods on the object under test rather than its collaborators. Perhaps I'll take a stab at seeing how that might change this test. If you're OK with it, though, I'm happy to hit the "merge" button.

**Comment by afeld** [comment]
> I'm wary of stubbing methods on the object under test rather than its collaborators. Perhaps I'll take a stab at seeing how that might change this test.

Would love suggestions!

**Comment by afeld** [comment]
...but this is consistent with the way I'm doing mocking in other parts of the plugin, so maybe we address that in a separate PR?

**Comment by mbland** [comment]
Oh, whoops, just saw your comment... If you don't like the commit I just pushed, I can revert/peel it off.

File: test/mapper.coffee [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    entity = null
+    orig_cwd = null
+    test_temp_dir = null
+
+    beforeEach ->
+      event = fixtures.getStartedEvent()
+      entity = event.entity
+      this.sinon.stub(client, 'call').callsArgWith(1, null, {}, {name: 'myorg'})
+      orig_cwd = process.cwd()
+      test_temp_dir = new temporary.Dir()
```
Comment (afeld) [inline test/mapper.coffee]: Minor: the convention in JS/CoffeeScript is to use camelCase for variables and properties/methods.

File: test/support/nocker.coffee [hunk 2]
```diff
@@ -0,0 +1,2 @@
+nock = require('nock')
+nock.disableNetConnect()
```
Comment (afeld) [inline test/support/nocker.coffee:2]: Hopefully this won't be necessary for much longer: https://github.com/pgte/nock/issues/324

**Comment by afeld** [comment]
@mbland I'm kind of indifferent...your way is more "pure", though takes more bookkeeping. Might also make things a little tricky once/if we start caching the config data, so that the data doesn't need to be read every time. I'm happy to cross that bridge when we come to it, though.

**Comment by mbland** [comment]
Yeah, I went with the temp dir method to make damn sure the environments were isolated between cases, but can strip out the temp dir stuff if you want.

**Pull request merged**